src/supplemental/nanolib/conf_ver2.c

```diff
@@ -351,6 +351,8 @@ conf_basic_parse_ver2(conf *config, cJSON *jso)
 			hocon_read_address_base(config, url, "bind", "nmq-tcp://", jso_tcp);
 			config->enable = true;
 			hocon_read_bool_base(config, enable, "enable", jso_tcp);
+		} else {
+			config->enable = false;
```

Comment:
if default value of `config->enable`  is false?

Reply:
No, if there is no configuration for `listeners.tcp`. Then `config->enable` will be false.